rbi/annotations/globalid.rbi

```diff
@@ -0,0 +1,6 @@
+# typed: strict
+
+class ActiveRecord::Base
+  # @shim: this is included at runtime https://github.com/rails/globalid/blob/5da635575f0f1f0251778dc21bf3b960e63b5791/lib/global_id/railtie.rb#L38
```

Comment:
I think we can point to the source of 1.0.0 rather than a specific commit?

https://github.com/rails/globalid/blob/v1.0.0/lib/global_id/railtie.rb#L38

Reply:
Sure!